docs/tutorials/typescript/subscriptions/index.md

```diff
@@ -1,416 +1,417 @@
 tags: [TypeScript, SDK]
 last_update:
-  date: 2021-10-01
+  date: 2024-06-07
 title: Build a subscription workflow with Temporal and TypeScript
-description: In this tutorial, we will tour all of the Workflow APIs you should know, primarily Signals, Queries, `condition`, and `sleep`, by building a realistic monthly subscription payments workflow that can be canceled and changed while it runs.
+description: Implement a subscription application using Temporal's Workflows, Activities, Signals, and Queries, enabling the payment workflow to be canceled or modified during execution.
 image: /img/temporal-logo-twitter-card.png
 ---
 
 ![Temporal TypeScript SDK](/img/sdk_banners/banner_typescript.png)
 
-import { OutdatedNotice } from '@site/src/components'
-
-<OutdatedNotice />
-
 ## Introduction
 
-In this tutorial, you will tour all of the [Workflow APIs](https://docs.temporal.io/typescript/workflows#workflow-apis) you should know, primarily Signals, Queries, `condition`, and `sleep` (and eventually Child Workflows and `continueAsNew`), by building a realistic monthly subscription payments workflow that can be canceled and changed while it runs.
-
-- The goal is to give you a more accessible introduction to these APIs by explaining them in context of a realistic application.
-- We also give an example of how you break down project requirements into Activities and Workflow logic.
-
-Your task is to write a Workflow for a limited time Subscription (eg a 36 month Phone plan) that satisfies these conditions:
-
-1. When the user signs up, **send a welcome email** and start a free trial for `TrialPeriod`.
-2. When the `TrialPeriod` expires, start the billing process
-   - If the user cancels during the trial, **send a trial cancellation email**.
-3. Billing Process:
-   - As long as you have not exceeded `MaxBillingPeriods`,
-   - **Charge the customer** for the `BillingPeriodChargeAmount`.
-   - Then wait for the next `BillingPeriod`.
-   - If the customer cancels during a billing period, **send a subscription cancellation email**.
-   - If Subscription has ended normally (exceeded `MaxBillingPeriods` without cancellation), **send a subscription ended email**.
-4. At any point while subscriptions are ongoing, be able to look up and change any customer's:
-   - Amount Charged
-   - Period number (for manual adjustments e.g. refunds)
-
-Of course, this all has to be fault tolerant, scalable to millions of customers, testable, maintainable, and observable.
+In this tutorial, you'll build a subscription application using Temporal and TypeScript. You'll use Temporal Workflows, Activities, sleep, Signals, and Queries to manage the core subscription logic. By leveraging Temporal's capabilities, you can handle complex subscription scenarios directly within Temporal's environment, reducing the need for external databases or queues. This reduces the complexity of the code you have to write and maintain.
```

Comment:
I think Sleep should be uppercase here. If it's not, I might make it last on the list (for example - "Activities, Signals, Queries, and sleep to manage the core subscription logic").  Could also say "eliminating the need for external databases" instead of reducing so reduces isn't repetitive in the paragraph.